frontend/test/pytest/test_callback.py

```diff
@@ -368,6 +368,28 @@ def f(x):
     f(arg)
 
 
+@pytest.mark.parametrize(
+    "arg",
+    [0.1, jnp.array(0.1)],
+)
+def test_dictionary(arg):
```

Comment:
Can we test arguments as well?

Reply:
It is already tested above.

Reply:
kwargs is a dictionary. There is a test for kwargs.

Reply:
Perfect 👍